finagle/buoyant/src/main/scala/com/twitter/finagle/buoyant/TlsClientConfig.scala

```diff
@@ -54,3 +54,14 @@ case class TlsClientConfig(
 }
 
 case class ClientAuth(certPath: String, keyPath: String)
+
+case class ClientTlsConfig(commonName: String, caCert: Option[String]) {
```

Comment:
Similar to the other comment, my reading of the code was that we were only supporting TLS for the thrift interpreter, so in adding http and mesh, it made sense to make all 3 the same. Open to discussion though.